internal/status/cache.go

```diff
@@ -53,8 +54,9 @@ type CacheEntry interface {
 type Cache struct {
 	proxyUpdates map[types.NamespacedName]*ProxyUpdate
 
-	gatewayRef   types.NamespacedName
-	routeUpdates map[types.NamespacedName]*ConditionsUpdate
+	gatewayRef     types.NamespacedName
+	gatewayUpdates map[types.NamespacedName]*GatewayConditionsUpdate
```

Comment:
remove this since it should already work.

Reply:
Same response as https://github.com/projectcontour/contour/pull/3805/files#r655899998.